src/main/java/org/s1ck/gdl/GDLLoader.java

```diff
@@ -338,6 +338,9 @@ public void enterVertex(GDLParser.VertexContext vertexContext) {
     Vertex v;
     if (variable != null && userVertexCache.containsKey(variable)) {
       v = userVertexCache.get(variable);
+      if (!getProperties(vertexContext.properties()).isEmpty()) {
+        throw new IllegalArgumentException("Properties for vertex " + variable + " defined multiple times");
```

Comment:
I think the text could be something like:
```
String.format("Vertex `%1$s` is declared multiple times. For reference, consider using `(%1$s)`.", variable)
```

Reply:
I chose to replace the second part with a more general sentence regarding entities.